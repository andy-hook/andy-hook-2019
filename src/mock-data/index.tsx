import { ISocialMeta } from "../types/model"

export const socialIcons: ISocialMeta = {
  twitter: {
    label: "twitter",
    url: "path/to/social/profile",
  },
  instagram: {
    label: "instagram",
    url: "path/to/social/profile",
  },
  linkedin: {
    label: "linkedin",
    url: "path/to/social/profile",
  },
  dribbble: {
    label: "dribbble",
    url: "path/to/social/profile",
  },
  github: {
    label: "github",
    url: "path/to/social/profile",
  },
}
