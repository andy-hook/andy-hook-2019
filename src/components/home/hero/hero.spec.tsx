import React from "react"
import renderer from "react-test-renderer"

import Hero from "./hero"
import { socialIcons } from "../../../mock-data"

describe("<Hero />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Hero socialIconData={socialIcons} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
