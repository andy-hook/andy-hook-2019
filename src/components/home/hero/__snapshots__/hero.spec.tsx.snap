// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> renders correctly 1`] = `
<div
  className="sc-htoDjs epVUox"
>
  <div
    className="sc-dnqmqq hvAzbm"
  >
    <h2
      className="sc-gzVnrw hXKoUt"
    >
      High-performance design & development
    </h2>
    <a
      className="sc-EHOje jZbwVB"
      href="path/to/social/profile"
      onClick={[Function]}
      target="_blank"
    >
      <span
        className="sc-bZQynM eNiIGl"
      >
        View my latest projects
      </span>
    </a>
  </div>
  <div
    className="sc-iwsKbI fmvhZb"
  >
    <div
      className="sc-htpNat hgHLpC"
    >
      <a
        aria-label="Get in touch"
        className="sc-bxivhb gcaUoY"
        href="mailto:hello@andy-hook.co.uk"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#mail"
            />
          </svg>
        </div>
      </a>
      <a
        aria-label="Twitter"
        className="sc-bxivhb gcaUoY"
        href="path/to/social/profile"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#twitter"
            />
          </svg>
        </div>
      </a>
      <a
        aria-label="Instagram"
        className="sc-bxivhb gcaUoY"
        href="path/to/social/profile"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#instagram"
            />
          </svg>
        </div>
      </a>
      <a
        aria-label="Linkedin"
        className="sc-bxivhb gcaUoY"
        href="path/to/social/profile"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#linkedin"
            />
          </svg>
        </div>
      </a>
      <a
        aria-label="Dribbble"
        className="sc-bxivhb gcaUoY"
        href="path/to/social/profile"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#dribbble"
            />
          </svg>
        </div>
      </a>
      <a
        aria-label="Github"
        className="sc-bxivhb gcaUoY"
        href="path/to/social/profile"
        onClick={[Function]}
        target="_blank"
      >
        <div
          className="sc-ifAKCX XzGje sc-bdVaJa dqBkPn"
        >
          <svg
            className="icon sc-bwzfXH kdIDyY"
          >
            <use
              xlinkHref="/icon-sprite.svg#github"
            />
          </svg>
        </div>
      </a>
    </div>
  </div>
  <div
    className="sc-gZMcBi kKRIiF"
  >
    <div
      className="sc-gqjmRU kFLaLw"
    />
    <div
      className="sc-VigVT lasMsM"
    >
      <test-file-stub
        className="sc-jTzLTM cZYbUl"
      />
    </div>
  </div>
</div>
`;
